src/common.js

```diff
@@ -31,3 +32,20 @@ export const permissions = [
     { label: _("Read, write and execute"), value: "7" },
     { label: _("Write and execute"), value: "3" },
 ];
+
+export const allowedtoRead = (selected, user) => {
```

Comment:
I'm not super thrilled about this function, but indeed it's cheap and an okay heuristic. Reading may still fail, though.

Reply:
I agree, but doing a stat on every file while showing the menu feels rather expensive. I'm open for better options

Reply:
It's ok, that was more thinking aloud. We need to handle errors, but that's discussed in a separate thread.